true/true.c

```diff
@@ -49,14 +55,16 @@ int
 main(int argc, char *argv[])
 {
 	int value;
+	value = get_true();
```

Comment:
This needs to be after cap_enter, we don't want to allow the system to be compromised by an exploit in get_true.